### PR TITLE
shalinee | TEST | correct invoice preview page design

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss [hunk 1]
```diff
@@ -580,7 +580,13 @@ span.icon-export-icon {
                justify-content: flex-start !important;
            }
         }
-
+        .invoice-modal-content{
+            left: 60px;
+            width: 100%;
+        }
+        .invoice-modal-content:after{
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss:587]: Consider using nesting and &:after in .invoice-modal

File: apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.scss [hunk 2]
```diff
@@ -1084,6 +1084,15 @@ strong {
         }
     }
 }
+@media(max-width:768px){
+    .invoice-modal-content{
+        left: 60px;
+        width: 100%;
+    }
+    .invoice-modal-content:after{
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.scss:1092]: Consider using nesting and &:after

File: apps/web-giddh/src/app/purchase/purchase-order-preview/purchase-order-preview.component.scss [hunk 3]
```diff
@@ -986,10 +987,21 @@ table.tax-table.item {
         padding-left: 30px !important;
     }
 }
+@media(max-width:768px){
+    .invoice-modal-content{
+        left: 60px;
+        width: 100%;
+    }
+    .invoice-modal-content:after{
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/purchase/purchase-order-preview/purchase-order-preview.component.scss:995]: Consider using nesting and &:after

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss:587]: Done

File: apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.scss:1092]: Done

File: apps/web-giddh/src/app/purchase/purchase-order-preview/purchase-order-preview.component.scss (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/purchase/purchase-order-preview/purchase-order-preview.component.scss]: Done

**Comment by shalinee-walkover** [comment]
https://app.clickup.com/t/2bbvtz

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**